File: serverSpring/src/main/java/com/vyatsu/auctionComplex/entity/api/ChatMessage.java
package com.vyatsu.auctionComplex.entity.api;

import com.fasterxml.jackson.annotation.JsonIgnore;
import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Entity
@Table(name = "chat_messages")
public class ChatMessage {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    @Getter
    @Setter
    private Integer id;

    @Column(name = "text", length = 250)
    @Getter
    @Setter
    private String text;

    @ManyToOne(fetch = FetchType.LAZY)
    @JsonIgnore
    @JoinColumn(name = "\"chat_id\"")
    @Getter
    @Setter
    private AuctionChat auctionChat;

    @ManyToOne(fetch = FetchType.LAZY)
    @JsonIgnore
    @JoinColumn(name = "\"UserId\"")
    @Getter
    @Setter
    private User user;

    @ManyToOne(fetch = FetchType.LAZY)
    @JsonIgnore
    @JoinColumn(name = "\"AttachmentId\"")
    @Getter
    @Setter
    private Attachment attachment;

    @Column(name = "\"timestamp\"", nullable = false)
    @Getter
    @Setter
    private Instant timestamp;

}

File: adminWF/Classes/ChatMessage.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace auctionComplex.Classes
{
    [Table("chat_messages")]
    internal class ChatMessage
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("text"), MaxLength(250)]
        public string Text { get; set; }

        public AuctionChat AuctionChat { get; set; }
        [Column("chat_id")]
        public int AuctionChatId { get; set; }

        public User User { get; set; }
        
        [Column("user_id")]
        public int UserId { get; set; }

        public Attachment Attachment { get; set; }
        [Column("attachment_id")]
        public int AttachmentId { get; set; }

        [Column("timestamp")]
        public DateTime Timestamp { get; set; }
    }
}
